diff --git a/chrome/browser/resources/side_panel/read_anything/BUILD.gn b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
index 7f2f7af6b2c6f0beecb069bc65dc16ccdd077a86..6bfd5ad6843acfc748537828dec56e8d7b409b33 100644
--- a/chrome/browser/resources/side_panel/read_anything/BUILD.gn
+++ b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
@@ -142,7 +142,7 @@ build_webui("build") {
             target_gen_dir) ]
 
   webui_context_type = "untrusted"
-  optimize = optimize_webui
+  optimize = false
   if (optimize) {
     optimize_webui_host =
         "chrome-untrusted://read-anything-side-panel.top-chrome"
