diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 4ce97be2fd64ec951e1ff99c3976a941ef06a3c7..751e1f3db6ba404ceb134244fac5a44e4638878d 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -5034,6 +5034,7 @@ constexpr char kEnterpriseIsolatedModeInternalName[] =
 
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
