diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 7d9990c9a6788c9410f5e807bb0f171a887ec659..4323fadedde503f95c7920aa4bbb7a2af5cc17d0 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -3352,6 +3352,7 @@ source_set("browser") {
       "//chrome/browser/safe_browsing/suspicious_site_warnings",
     ]
   }
+  public_deps += [ "//brave/browser:core" ]
 
   # This target intentionally has no sources of its own and no
   # allow_circular_includes_from; it only aggregates dependencies. Do NOT add
@@ -3521,6 +3522,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
