diff --git a/chrome/browser/extensions/chrome_extensions_browser_client.cc b/chrome/browser/extensions/chrome_extensions_browser_client.cc
index 18ed6069c68aafdc8959c1dc57740791303d3d0f..2f2e1cace985bddaa8585470590343f38512a8d5 100644
--- a/chrome/browser/extensions/chrome_extensions_browser_client.cc
+++ b/chrome/browser/extensions/chrome_extensions_browser_client.cc
@@ -275,7 +275,7 @@ bool RegisterTransformers() {
 ChromeExtensionsBrowserClient::ChromeExtensionsBrowserClient()
     : resource_manager_(
           std::make_unique<ChromeComponentExtensionResourceManager>()),
-      api_client_(std::make_unique<ChromeExtensionsAPIClient>()),
+      api_client_(std::make_unique<BraveExtensionsAPIClient>()),
       event_router_forwarder_(base::MakeRefCounted<EventRouterForwarder>()) {
   AddAPIProvider(std::make_unique<CoreExtensionsBrowserAPIProvider>());
   AddAPIProvider(std::make_unique<ChromeExtensionsBrowserAPIProvider>());
@@ -690,7 +690,7 @@ ChromeExtensionsBrowserClient::CreateUpdateClientConfigurator(
       override_url = update_url;
     }
   }
-  return ChromeUpdateClientConfig::Create(context, override_url);
+  return BraveUpdateClientConfig::Create(context, override_url);
 }
 
 std::unique_ptr<ScopedBrowserContextKeepAlive>
