diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
index 411d05fc91a5e9701dc84b29fb6babccaf4ece27..970a7e4ef545e7513fc6c1061b5ab9ab72423813 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
@@ -981,6 +981,14 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
 
                         private void closeIfNoTabsAndHomepageEnabled(
                                 boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
+                            if (getTabModelSelector().getTotalTabCount() == 0) {
+                                shouldRemoveWindowWithZeroTabs = shouldRemoveWindowWithZeroTabs && org.chromium.chrome.browser.partnercustomizations.CloseBraveManager.shouldCloseAppWithZeroTabs();
+                            }
+                            closeIfNoTabsAndHomepageEnabled_ChromiumImpl(isPendingClosure, shouldRemoveWindowWithZeroTabs);
+                        }
+
+                        private void closeIfNoTabsAndHomepageEnabled_ChromiumImpl(
+                                boolean isPendingClosure, boolean shouldRemoveWindowWithZeroTabs) {
                             if (getTabModelSelector().getTotalTabCount() == 0) {
                                 if (shouldRemoveWindowWithZeroTabs) {
                                     mMultiInstanceManager.closeChromeWindowIfEmpty(mWindowId);
@@ -3921,6 +3929,11 @@ public class ChromeTabbedActivity extends ChromeActivity implements PreAttachInt
                 new TabModelSelectorObserver() {
                     @Override
                     public void onTabStateInitialized() {
+                        onTabStateInitialized_ChromiumImpl();
+                        onTabStateInitializedHandler();
+                    }
+
+                    private void onTabStateInitialized_ChromiumImpl() {
                         if (mMultiInstanceManager != null) {
                             mMultiInstanceManager.onTabStateInitialized();
                         }
