diff --git a/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java
index ca608f79288de531ea38fe510ec18210b01f6c9b..dfc5f78faab18a195ad0e66904ca2526a7bb501e 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/settings/search/SettingsSearchCoordinator.java
@@ -4,6 +4,8 @@
 
 package org.chromium.chrome.browser.settings.search;
 
+import org.chromium.chrome.browser.theme.BraveDynamicColors;
+import org.chromium.components.browser_ui.settings.BraveSettingsIconTintUtils;
 import static androidx.annotation.VisibleForTesting.PRIVATE;
 
 import static org.chromium.base.CallbackUtils.emptyRunnable;
@@ -309,6 +311,12 @@ public class SettingsSearchCoordinator
     /** Initializes search UI, sets up listeners, backpress action handler, etc. */
     @Initializer
     public void initializeSearchUi(@Nullable Bundle savedState) {
+        initializeSearchUi_ChromiumImpl(savedState);
+        View searchBox = requireViewById(R.id.search_box);
+        BraveSettingsIconTintUtils.applyIconTint(searchBox.requireViewById(R.id.search_icon), BraveDynamicColors.isDynamicColorsEnabled());
+    }
+
+    private void initializeSearchUi_ChromiumImpl(@Nullable Bundle savedState) {
         mUseMultiColumn = mUseMultiColumnSupplier.getAsBoolean();
         // AppBarLayout(app_bar_layout)
         //         +-- FrameLayout
