diff --git a/components/heap_profiling/in_process/heap_profiler_parameters.cc b/components/heap_profiling/in_process/heap_profiler_parameters.cc
index f2fb11c809ad97dbf3114b03bb5e17cb84edcc92..a0ab20031174d90805a4c17ff266894727e529aa 100644
--- a/components/heap_profiling/in_process/heap_profiler_parameters.cc
+++ b/components/heap_profiling/in_process/heap_profiler_parameters.cc
@@ -125,7 +125,8 @@ constexpr base::FeatureParam<double> kUtilityHashSetLoadFactor{
 
 }  // namespace
 
-BASE_FEATURE(kHeapProfilerReporting, base::FEATURE_ENABLED_BY_DEFAULT);
+// kHeapProfilerReporting feature state is enforced via plaster rewrite.
+BASE_FEATURE(kHeapProfilerReporting, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kHeapProfilerChurnReporting, base::FEATURE_DISABLED_BY_DEFAULT);
 
 const base::FeatureParam<double> kHeapChurnSubsamplingChance{
