diff --git a/components/contextual_tasks/public/features.cc b/components/contextual_tasks/public/features.cc
index c15eb7588175c2fc1d372e700ed415e23744955d..119e19bece046d3054513c03729102e66b375956 100644
--- a/components/contextual_tasks/public/features.cc
+++ b/components/contextual_tasks/public/features.cc
@@ -35,6 +35,7 @@ std::optional<bool> g_sticky_conversation_enabled_override;
 namespace contextual_tasks {
 
 // Enables the contextual tasks side panel while browsing.
+// kContextualTasks feature state is enforced via plaster rewrite.
 BASE_FEATURE(kContextualTasks, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE(kContextualTasksPrivateApiNoAnimation,
              base::FEATURE_DISABLED_BY_DEFAULT);
@@ -181,6 +182,7 @@ BASE_FEATURE(kContextualTasksOverrideShowBottomSheetOnLargeScreen,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
 // Enables prefetching of cookies for contextual tasks.
+// kContextualTasksCookiePrefetch feature state is enforced via plaster rewrite.
 BASE_FEATURE(kContextualTasksCookiePrefetch, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kAimTriggeredThreadLinks, base::FEATURE_ENABLED_BY_DEFAULT);
