diff --git a/components/autofill/core/browser/suggestions/suggestion.h b/components/autofill/core/browser/suggestions/suggestion.h
index 8ec85b5309c70bf5ec9a38444d8106f0c147b5fc..c052d3d421dfa0e3f35698d61fdede3610e3f197 100644
--- a/components/autofill/core/browser/suggestions/suggestion.h
+++ b/components/autofill/core/browser/suggestions/suggestion.h
@@ -730,6 +730,8 @@ struct Suggestion {
 
   // Returns whether the user is able to focus or select the suggestion.
   bool IsSelectable() const;
+  bool brave_new_email_alias_suggestion = false;
+  bool brave_email_alias_suggestion = false;
 };
 
 void PrintTo(const Suggestion& suggestion, std::ostream* os);
