diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index fc72f43ec09e4aec93ca00e91d2923ad4deb6ca0..fd7a8ab00efa44b10541c424be746251fe7a6c2b 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3633,6 +3633,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
