diff --git a/DEPS b/DEPS
index 290cb0ac683b9cc8ce9139b301fbf9e6b92079d9..ce1620f0fa56a416a0a1894881348f8ebbc42db8 100644
--- a/DEPS
+++ b/DEPS
@@ -4532,7 +4532,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
