diff --git a/net/base/features.cc b/net/base/features.cc
index 0f461e88baa38e6cf2221a8bcb717a0cb7fa7a51..5a5be424756ee7b2b8da3787386c8b499b8740b8 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -218,8 +218,9 @@ BASE_FEATURE(kSplitCacheByNetworkIsolationKey,
 BASE_FEATURE(kSplitCodeCacheByNetworkIsolationKey,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
+// kPartitionConnectionsByNetworkIsolationKey feature state is enforced via plaster rewrite.
 BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kSplitHostCacheByNetworkAnonymizationKey,
              base::FEATURE_ENABLED_BY_DEFAULT);
@@ -437,6 +438,7 @@ BASE_FEATURE(kForceThirdPartyCookieBlocking,
 
 BASE_FEATURE(kEnableEarlyHintsOnHttp11, base::FEATURE_DISABLED_BY_DEFAULT);
 
+// kEnableWebTransportDraft07 feature state is enforced via plaster rewrite.
 BASE_FEATURE(kEnableWebTransportDraft07, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kWebTransportFineGrainedThrottling,
@@ -751,7 +753,8 @@ BASE_FEATURE(kRestrictAbusePorts, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kRestrictAbusePortsOnLocalhost, base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kTLSTrustAnchorIDs, base::FEATURE_ENABLED_BY_DEFAULT);
+// kTLSTrustAnchorIDs feature state is enforced via plaster rewrite.
+BASE_FEATURE(kTLSTrustAnchorIDs, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kNonMtcTrustAnchorIDs, base::FEATURE_ENABLED_BY_DEFAULT);
 
