diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
index 7c9ae72dc4064f3c16aa8b6d9f9bba4aa7806c23..ab81a5eca48958475e46a13576b1c8a1919ec346 100644
--- a/third_party/tflite/BUILD.gn
+++ b/third_party/tflite/BUILD.gn
@@ -396,6 +396,7 @@ if (!use_litert_tflite) {
 
     visibility = [
       ":tflite_benchmark",
+      "//brave/services/passage_embeddings/*",
       "//components/optimization_guide/internal/*",
       "//modules/*",
       "//services/webnn/*",
